### PR TITLE
termius: Update to version 7.21.2

### PR DESCRIPTION
**Pull request by igitur**

### CODE REVIEW

File: bucket/termius.json [hunk 1]
```diff
@@ -21,8 +21,8 @@
         ]
     ],
     "checkver": {
-        "url": "https://autoupdate.termius.com/win/latest.yml",
-        "regex": "version:\\s+([\\d.]+)"
+        "url": "https://docs.termius.com/changelog/desktop",
+        "regex": "<span data-offset-key=\"[^\"]+\">\\s*(\\d+\\.\\d+\\.\\d+) -"
```
Comment (Ash258) [inline bucket/termius.json:25]: Test before applying

```suggestion
        "regex": "\">\\s*([\\d.]+)\\s+-\\s+\\d+"
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/termius.json (hunk 1 above)
Comment (igitur) [reply bucket/termius.json:25]: that regex is wrong

Comment (Ash258) [reply bucket/termius.json:25]: > Regex is wrong

Lie.

image

File: bucket/termius.json [hunk 2]
```diff
@@ -21,11 +21,11 @@
         ]
     ],
     "checkver": {
-        "url": "https://autoupdate.termius.com/win/latest.yml",
-        "regex": "version:\\s+([\\d.]+)"
+        "url": "https://autoupdate.termius.com/windows/latest.yml",
```
Comment (igitur) [inline bucket/termius.json:24]: How did you find this URL?

**Pull request merged**